Bulldozer.CSV/HelperMethods.cs

```diff
@@ -0,0 +1,502 @@
+                //        bool addressAlreadyExsistsExactMatch = groupLocations.Where( a =>
+                //            a.GroupLocationTypeValueId == groupAddressImport.GroupLocationTypeValueId
+                //            && (
+                //                 a.Location.Street1 == groupAddressImport.Street1
+                //                    && a.Location.Street2 == groupAddressImport.Street2
+                //                    && a.Location.City == groupAddressImport.City
+                //                    && a.Location.County == groupAddressImport.County
+                //                    && a.Location.State == groupAddressImport.State
+                //                    && a.Location.Country == groupAddressImport.Country
+                //                    && a.Location.PostalCode == groupAddressImport.PostalCode
+                //                 ) ).Any();
+
+                //        if ( !addressAlreadyExsistsExactMatch )
+                //        {
+                //            var locationService = new LocationService( rockContextForGroupUpdate );
+
+                //            Location location = locationService.Get( groupAddressImport.Street1, groupAddressImport.Street2, groupAddressImport.City, groupAddressImport.State, groupAddressImport.PostalCode, groupAddressImport.Country, false );
+
+                //            if ( !groupLocations.Where( a => a.GroupLocationTypeValueId == groupAddressImport.GroupLocationTypeValueId && a.LocationId == location.Id ).Any() )
+                //            {
+                //                var groupLocation = new GroupLocation();
+                //                groupLocation.GroupId = group.Id;
+                //                groupLocation.GroupLocationTypeValueId = groupAddressImport.GroupLocationTypeValueId;
+                //                groupLocation.IsMailingLocation = groupAddressImport.IsMailingLocation;
+                //                groupLocation.IsMappedLocation = groupAddressImport.IsMappedLocation;
+
+                //                if ( location.GeoPoint == null && groupAddressImport.Latitude.HasValue && groupAddressImport.Longitude.HasValue )
+                //                {
+                //                    location.SetLocationPointFromLatLong( groupAddressImport.Latitude.Value, groupAddressImport.Longitude.Value );
+                //                }
+
+                //                groupLocation.LocationId = location.Id;
+                //                groupLocationService.Add( groupLocation );
+
+                //                addressesUpdated = true;
+                //            }
+                //        }
+                //    }
+                //}
+
+                // update schedule
+                bool scheduleUpdated = false;
+                DayOfWeek meetingDay;
+                if ( groupImport.MeetingDay.IsNullOrWhiteSpace() && Enum.TryParse( groupImport.MeetingDay, out meetingDay ) )
```

Comment:
Doesn't look right. This condition will never be true. need to evaluation what intent is.